### PR TITLE
George format fixes

### PR DESCRIPTION
**Pull request by Georgehe4**
@devoncarew 
Fixing formatting issues.

### CODE REVIEW

**Comment by devoncarew** [comment]
Lgtm

**Comment by Georgehe4** [comment]
Also added the dialog changes in this PR.

screen shot 2015-07-14 at 3 01 43 pm

File: lib/dialogs.dart [hunk 1]
```diff
@@ -85,26 +80,49 @@ class EmbedDialog extends DDialog {
...
     Uri url = Uri.parse(window.location.toString());
     String home = url.host;
-    _doc.add(new SpanElement()..text = 'Dart + Documentation: ' ..style.paddingRight = "12px");
-    _doc.add(new InputElement()..value = '${home}/embed-dart.com?id=${gist.id}' ..attributes['flex'] = '');
-    _html.add(new SpanElement()..text = "Dart + Html: " ..style.paddingRight = "12px");
-    _html.add(new InputElement()..value = '${home}/embed-html.com?id=${gist.id}' ..attributes['flex'] = '');
-    _inline.add(new SpanElement()..text = "Dart (Minimal): " ..style.paddingRight = "12px");
-    _inline.add(new InputElement()..value = '${home}/embed-inline.com?id=${gist.id}' ..attributes['flex'] = '');
+    _doc.add(new SpanElement()
+      ..text = 'Dart + Documentation: '
```
Comment (devoncarew) [inline lib/dialogs.dart]: Can you instead add a class to this, and have the class have the padding and min-width settings? The class would probably need to be defined in web/index.css.

File: lib/dialogs.dart [hunk 2]
```diff
@@ -85,26 +80,49 @@ class EmbedDialog extends DDialog {
...
+      ..attributes['flex'] = '');
+    _inline.add(new SpanElement()
+      ..text = 'Dart (Minimal): '
+      ..style.paddingRight = '12px'
+      ..style.minWidth = '140px');
+    _inline.add(new InputElement()
+      ..value = '${home}/embed-inline.html?id=${gist.id}'
+      ..attributes['flex'] = '');
+    _info.add(new SpanElement()
+      ..text = 'Need more control? Checkout out our '
```
Comment (devoncarew) [inline lib/dialogs.dart]: `Checkout` ==? `Check out`

**Comment by devoncarew** [comment]
lgtm

**Comment by Georgehe4** [comment]
Merging

**Pull request merged**